### DISCUSSION TITLE
Precursors mzWindow

### DISCUSSION THREAD

**Discussion by PSobrevalsAlcaraz** [vdemichev/DiaNN#225] [Q&A]
Hello Vadim,

I have been doing several analyses with DiaNN, and now I would like to dive in a little bit more and be able to see the peptides in its own scan. I was wondering if there's any option to return the precursors found in each MS1 scan? I am trying to match MS2s to MS1s and I can't get to identify which peaks are from peptides (and isotopes) and which ones come from noise.... Find picture of example. 
So I was wondering if there's any output file that gives this information or if there's a "easy" way to find it from any of the output files that you may know?

Thank you so much for your time and for developing such a nice and useful tool :)
![MS1_scanWindow](https://user-images.githubusercontent.com/67247574/139271488-58ff47c6-4058-4023-b4dc-3842e7547f24.png)

Paula

**Comment by vdemichev** [comment]
Hi Paula.

In DIA precursors are matched to 'peak groups' which are sets of chromatograms in the vicinity of the peak apex. So each precursor is matched to multiple consecuitive spectra. DIA-NN records the apex retention time & the RT window used for quantification, so these can be used to determine which spectra have been matched to.

Best,
Vadim
